health/micro-ui-react19/web/packages/css/src/pages/employee/commodityManagement.scss

```
@@ -0,0 +1,491 @@
+  border-top: 1px solid #D6D5D4;
+  font-size: 13px;
+  color: #505A5F;
+}
+
+.commodity-management-inbox-table-wrapper{
+.commodity-management-inbox-table {
+  .rdt_TableBody {
+    border: none !important;
+    border-bottom: 0.063rem solid var(--digitv2-lightTheme-divider) !important;
+  }
+  overflow-x: auto !important;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)